src/swift/Wrapper.swift

```diff
@@ -307,9 +323,9 @@ internal enum _OSQoSClass : UInt32  {
 		case 0x21: self = .QOS_CLASS_USER_INTERACTIVE
 		case 0x19: self = .QOS_CLASS_USER_INITIATED
 		case 0x15: self = .QOS_CLASS_DEFAULT
-		case 0x11: self = QOS_CLASS_UTILITY
-		case 0x09: self = QOS_CLASS_BACKGROUND
-		case 0x00: self = QOS_CLASS_UNSPECIFIED
+		case 0x11: self = .QOS_CLASS_UTILITY
+		case 0x09: self = .QOS_CLASS_BACKGROUND
+		case 0x00: self = .QOS_CLASS_UNSPECIFIED
```

Comment:
not related to primary DispatchData changes; fixing compiler warning (will be an error in Swift3).